components/common/src/test/java/com/hotels/styx/api/ResponseEventListenerTest.java

```diff
@@ -53,97 +49,96 @@ public void setUp() {
 
     @Test
     public void doesntFireUnnecessaryEvents() {
```

Comment:
Could we include some description (either in the name or a comment) of what makes these events unnecessary? Is it just saying that the events being fired never happened, e.g.
```suggestion
    public void doesntFireEventsThatNeverOccurred() {
```
?